src/jit/gentree.cpp

```diff
@@ -7122,7 +7106,8 @@ GenTreePtr GenTree::GetChild(unsigned childNum)
     printf("%c", (flags & GTF_CALL          ) ? 'C' : '-');
     printf("%c", (flags & GTF_EXCEPT        ) ? 'X' : '-');
     printf("%c", (flags & GTF_GLOB_REF      ) ? 'G' : '-');
-    printf("%c", (flags & GTF_ORDER_SIDEEFF ) ? 'O' : '-');
+    printf("%c", (flags & GTF_MORPHED       ) ? '+' :
+                 (flags & GTF_ORDER_SIDEEFF ) ? 'O' : '-');
```

Comment:
I'm not sure I understand what you are doing here - are you simply trying to preserve the alignment of bits, or is this really some sort of tri-state, where we have only one of GTF_MORPHED or GTF_ORDER_SIDEEFF, or neither? Or does GTF_ORDER_SIDEEFF imply GTF_MORPHED? This is weird enough to deserve a comment.